examples/fastai/READMD.md

```diff
@@ -0,0 +1,42 @@
+### MLflow Fastai Example
+
+This simple example illustrates how you can use `mlflow.fastai.autolog()` API
+to track parameters, metrics, and artifacts while training a simple MNIST model. Derived from fastai GitHub Repository 
+of [vision examples](https://github.com/fastai/fastai/blob/master/examples/train_mnist.py), the code is modified to run 
+as a MLflow project, with either default or supplied arguments. The default arguments are learning rate(`lr=0.01`) 
+and number of epochs (`epochs=5`).
+
+You can use this example as a template and attempt advanced examples in
+[Fastai tutorials](https://docs.fast.ai/vision.html), using `mlfow.fastai` model flavor and MLflow tracking API to
+track your experiments.
+
+#### How to run the code
```

Comment:
```suggestion
#### How to run this code
```

Reply:
Or just `How to run` might suffice.

Reply:
Isn't this obvious what code we running?